### PR TITLE
postgresqlPackages.pgaudit: init at 1.7.0

### PR DESCRIPTION
**Pull request by idontgetoutmuch**
###### Description of changes

Audit logging extension for Postgres: https://github.com/pgaudit/pgaudit

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by idontgetoutmuch** [comment]
@balsoft as discussed, if you could take a look that would be great

**Comment by balsoft** [comment]
Please, make sure your PR name and commit messages conform to CONTRIBUTING.md: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes

It should be along the lines of `postgresqlPackages.pgaudit: init at 1.5.1`

**Review by balsoft — APPROVED** [review]
Diff LGTM

File: pkgs/servers/sql/postgresql/ext/pgaudit.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, kerberos, openssl, postgresql }:
```
Comment (ajs124) [inline pkgs/servers/sql/postgresql/ext/pgaudit.nix:1]:
```suggestion
{ lib, stdenv, fetchFromGitHub, libkrb5, openssl, postgresql }:
```
`kerberos` is an alias, so this won't evaluate without them

File: pkgs/servers/sql/postgresql/ext/pgaudit.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "pgaudit";
+    repo = "pgaudit";
+    rev = version;
+    sha256 = "sha256-5z2F5huydSX9kSEFGTXyAw0x9mHM9+4bFqMn4jS+sG4=";
+  };
+
+  buildInputs = [ kerberos openssl postgresql ];
```
Comment (ajs124) [inline pkgs/servers/sql/postgresql/ext/pgaudit.nix:14]:
```suggestion
  buildInputs = [ libkrb5 openssl postgresql ];
```
same here

**Review by ajs124 — CHANGES_REQUESTED** [review]

**Pull request merged**